File: octopoints-kotlin/app/src/main/java/dev/tpavani/octopoints/service/utils/GetListFilter.kt
package dev.tpavani.octopoints.service.utils

class GetListFilter {
    var id: Int? = null
    var active: Boolean? = null

    fun setId(id: Int): GetListFilter{
        this.id = id
        return this
    }

    fun setActive(active: Boolean): GetListFilter{
        this.active = active
        return this
    }
}

File: octopoints_flutter/lib/service/utils/GetListFilter.dart
class GetListFilter {
  int? id;
  bool? isActive;

  GetListFilter setId(int? id){
    this.id = id;
    return this;
  }

  GetListFilter setIsActive(bool isActive){
    this.isActive = isActive;
    return this;
  }
}